### PR TITLE
Created ReadMe markdown.

### PR DESCRIPTION
**Pull request by jeremy-jewett**
---

[Related Trello Card](https://trello.com/c/ZOQ4Zh5E)

---

### Location:

-app/tests/README.md

---

### Details:

1. Created README for future developers

---

### Committed Features:
1. Creation of file

---

### Any Issues:
1. None

### CODE REVIEW

File: app/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+can be done by running the following command:
+
+`python -m unittest discover`
+
+The "discover" part of the command line will discover all the available tests, and run
+all of them.  These tests can be run individually if you would like.  Documentation on
+the different modifications to this command can be found in the documentation for
+unittest, which lies just below.
+
+[Documentation for Unittest](https://docs.python.org/3/library/unittest.html)
```
Comment (jmmiddour) [inline app/tests/README.md:22]: Just need a space here to separate the links for the documentation. Otherwise, looks great.

**Review by jmmiddour — CHANGES_REQUESTED** [review]

**Review by jmmiddour — APPROVED** [review]
Everything looks good and is ready to be merge.

**Review by ToddDiFronzo — APPROVED** [review]
Informative and concise

**Review by jrslagle — APPROVED** [review]

**Review by jproctor-rebecca — APPROVED** [review]
Thank you for resolving the previous comments.

**Review by ike-okonkwo — APPROVED** [review]
Testing suite documentation

**Pull request merged**